backend/de.metas.manufacturing/src/main/sql/postgresql/system/30-de.metas.adempiere.libero/5659000_sys_gh13716_PP_Product_Planning_drop_MaxManufacturedQty.sql

```diff
@@ -0,0 +1,114 @@
+/*
+ * #%L
+ * de.metas.manufacturing
+ * %%
+ * Copyright (C) 2022 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+-- 2022-10-05T14:15:56.091Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+DELETE FROM AD_UI_Element WHERE AD_UI_Element_ID=575353
```

Comment:
as I understood from @metasnw  we shall not drop the PP_Product_Planning.MaxManufacturedQtyPerOrder but we shall fallback to it in case there is no qty/cycle specified at resource level.

Reply:
ah.. alright, for me it sounded like it should be gone, we'll put it back and use it as fallback 👍

Reply:
@cp-ps  not needed anymore (check requirements changes)